sable_ircd/src/server/mod.rs

```diff
@@ -76,6 +79,9 @@ pub struct ClientServer {
 
     node: Arc<NetworkNode>,
     listeners: Movable<ListenerCollection>,
+
+    // Any general static info (responses for MOTD, ADMIN, and so on)
+    pub infos: Infos,
```

Comment:
Same as above - a more descriptive name would be nice

Reply:
Per chat, now info_strings